objectstore-service/src/lib.rs

```diff
@@ -91,15 +109,37 @@ impl StorageService {
         &self,
         key: &ScopedKey,
     ) -> anyhow::Result<Option<(Metadata, BackendStream)>> {
-        self.0.backend.get_object(key).await
+        match key.key.backend {
+            1 => self.0.small_backend.get_object(key).await,
+            2 => self.0.large_backend.get_object(key).await,
+            _ => anyhow::bail!("invalid backend"),
+        }
```

Comment:
_nit_: Can we refactor this into a helper function and introduce consts for the magic numbers?